src/engine/database/mod.rs

```diff
@@ -18,12 +18,13 @@ use bb8_bolt::BoltConnectionManager;
 #[cfg(any(feature = "cosmos", feature = "gremlin"))]
 use gremlin_client::GremlinClient;
 use std::collections::HashMap;
+use std::convert::TryFrom;
 #[cfg(any(feature = "cosmos", feature = "gremlin", feature = "neo4j"))]
 use std::env::var_os;
 use std::fmt::Debug;
 
 #[cfg(feature = "gremlin")]
-fn env_bool(var_name: &str) -> Result<bool, Error> {
+pub fn env_bool(var_name: &str) -> Result<bool, Error> {
```

Comment:
@jeffrey - is there a way to access this in `tests` without making it `pub`? I tried `pub(crate)` but it didn't work.

Reply:
I don't think so.  Integration tests are separate crates from the system under test.